site/src/components/scroll.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Also, should it be in the components folder? I feel like the component is something complex, with additional helpers created specifically for it 🤔 I think Atomics is a better place, but I'm not 100% sure

Reply:
And I realized there are no more `overflow-x-hidden` nor overflow-y-hidden` helpers, I assume you removed these two assuming people would use` scroll-horizontal` \ `scroll-vertical` instead, so I think it would be good to keep that info close to each other.

Reply:
Yep, I think these are both components. Agreed that they are simple, but components need not be complex. Atomics are tied whenever possible to a single CSS property (or maybe two if we're being a little forgiving). That properties name is reflected by CSS. The name alone here is a clue that these aren't Atomics as we've defined them until now.